public/video-ui/src/util/parseGridMetadata.js

```diff
@@ -0,0 +1,33 @@
+function parseMimeType(mimeType) {
+
+  //Normalise Mime Types coming from the grid.
+  switch(mimeType) {
+    case "jpg":
+      return "image/jpeg";
+    case "png":
+      return "image/png";
+  }
+
+  return mimeType;
+}
+
+
+function parseAsset(asset) {
+  return {
+    url: asset.secureUrl,
+    mimeType: parseMimeType(asset.mimeType),
+    size: asset.size,
```

Comment:
the YouTube API has a limit of [2MB](https://developers.google.com/youtube/v3/docs/thumbnails/set), should we add some validation here too?